sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample10_SecretReference.md

```diff
@@ -0,0 +1,50 @@
+# Create, Retrieve and Delete a Secret Reference
+
+The AppConfiguration service supports KeyVault Secret reference settings.
+Secret references are settings that follow specific JSON schema for the `Value`, and the `application/vnd.microsoft.appconfig.keyvaultref+json;charset=utf-8` content type.
+The `Azure.Data.AppConfiguration` library provides a strongly-typed way of managing Secret references.
+This sample shows how to use the library to create, retrieve, and delete Secret references.
+
```

Comment:
Samples usually start with a [Create client](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample1_HelloWorld.md#create-a-configurationclient) section

Reply:
Do you think it's worth duplicating the same section everywhere?

Reply:
The idea to have it in each file is that the user won't need to jump across files to have basic behavior.
I guess to your point, this new samples are advanced samples so might not be needed.